src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DsesFilterAndTransform.cs

```diff
@@ -404,9 +404,6 @@ public void Dispose()
         }
     }
 
-    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2112:ReflectionToRequiresUnreferencedCode",
```

Comment:
@sbomer @agocke - CI was previously failing with https://github.com/dotnet/runtime/pull/104134/checks?check_run_id=27545810735

> /_/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DsesFilterAndTransform.cs(1005,9): error IL2121: (NETCORE_ENGINEERING_TELEMETRY=Build) System.Diagnostics.DsesFilterAndTransform.MakeImplicitTransforms(Type): Unused 'UnconditionalSuppressMessageAttribute' for warning 'IL2112'. Consider removing the unused warning suppression.


Do you know why these warnings weren't happening before the code was moved out to separate files?